components/scream/src/physics/p3/atmosphere_microphysics.cpp

```diff
@@ -1,6 +1,8 @@
-#include "physics/p3/scream_p3_interface.hpp"
 #include "physics/p3/atmosphere_microphysics.hpp"
 #include "physics/p3/p3_inputs_initializer.hpp"
+#include "physics/p3/p3_main_impl.hpp"
+// Needed for p3_init, the only F90 code still used.
+#include "physics/p3/p3_f90.hpp"
```

Comment:
TBH I don't know.  My understanding is that it sets up the table lookup values used within p3_main.  @jgfouca , does that sound correct?

Reply:
It doesn't seem to write in any C++ provided ptr though (or in any array that is manually copied to a C++ ptr). My guess is that we don't need it. Have you tried to not call the f90 stuff and see what happens?

Reply:
@bartgol , @AaronDonahue , I haven't thought about this in a while, but my recollection is we didn't want to bother converting initialization since we don't care about it's performance. p3_init must be called in order for various tables to be read and initialized.

Reply:
TBC, I'm ok keeping initialization in F90. What puzzles me is that I can't see where the data is transferred from F90 to C. But if you know it is, than we're good.